cluster/get-kube.sh

```diff
@@ -61,16 +61,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# TODO(pipejakob): Use dl.k8s.io/bazel once the redirection uses the correct bucket.
+KUBERNETES_BAZEL_RELEASE_URL="${KUBERNETES_BAZEL_RELEASE_URL:-${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release-dev/bazel}}"
```

Comment:
Yeah, there's a short explanation in the TODO comment, but to elaborate: right now, since `/bazel` is brand new, there is no explicit support for it in our dl.k8s.io handler, so visiting https://dl.k8s.io/bazel/blah incorrectly redirects to our normal release bucket (`kubernetes-release`) instead of the CI bucket (`kubernetes-release-dev`). There's an explicit pattern match so that https://dl.k8s.io/ci/blah redirects to our CI bucket, and I would like to add another one so that `/bazel/*` points to this as well. I wanted this PR to happen first to vet the idea of fetching these bazel versions, but if this gets merged, then I'll update our dl.k8s.io handler, and then update this code again to use the vanity URL after it gets deployed.